testing/srt-test-live.cpp

```diff
@@ -761,6 +762,17 @@ int main( int argc, char** argv )
         }
     }
 
+    string retryphrase = Option<OutString>(params, "", o_retry);
+    if (retryphrase != "")
+    {
+        if (retryphrase[retryphrase.size()-1] == 'a')
+        {
+            transmit_retry_always = true;
+            retryphrase = retryphrase.substr(0, retryphrase.size()-1);
+        }
+
+        transmit_retry_connect = stoi(retryphrase);
+    }
```

Comment:
Added a `-retry` option that stubbornly tries to reconnect when the connection was broken. The argument is a number of times to retry, -1 is unlimited, when `a` is added to the number, it means it should retry no matter what the reason was (otherwise it only retries if it was timeout).